app/controllers/users_controller.rb

```diff
@@ -128,8 +128,8 @@ def output
     @day = calculate_day
     daystart = @day.beginning_of_day
     dayend = @day.end_of_day
-    @posts = Post.where(user: current_user).where('created_at between ? AND ?', daystart, dayend).pluck(:content)
-    @replies = Reply.where(user: current_user).where('created_at between ? AND ?', daystart, dayend).pluck(:content)
+    @posts = Post.where(user: current_user).where('created_at between ? AND ?', daystart, dayend).ordered_by_id.pluck(:content)
+    @replies = Reply.where(user: current_user).where('created_at between ? AND ?', daystart, dayend).order(post_id: :asc).ordered.pluck(:content)
```

Comment:
This probably won't intersperse properly with @posts, do we care given someday post-to-reply?

Reply:
yeah I figured trying to make posts intersperse wasn't worth it here given posts-to-replies Hopefully Eventually